connexion/decorators/parameter.py

```diff
@@ -49,20 +51,27 @@ def wrapper(*args, **kwargs):
         logger.debug('Function Arguments: %s', arguments)
 
         try:
-            body_parameters = flask.request.json or {}  # type: dict
+            body_parameters = flask.request.json or {}
         except exceptions.BadRequest:
             body_parameters = {}
 
         # Add body parameters
-        for key, value in body_parameters.items():
-            if key not in arguments:
-                logger.debug("Body parameter '%s' not in function arguments", key)
+        if isinstance(body_parameters, dict):
```

Comment:
When connexion receives a request with type ``object`` body_parameters is a dict and I was getting the keys from inside that object and passing them to the function as arguments. Do you think it would be better to provide the body as a single argument in this case?